### DISCUSSION TITLE
Educational Benefits NOT started Yet

### DISCUSSION THREAD

**Discussion by NagbhushanPai** [community/community#129193] [GitHub Education]
### Hi everyone,

My Benefits were Approved 3 days ago on 17th June 2024, yet I haven't received any e-mails yet. Kindly let me know how can i gontact Github officials to address my grievances.

**Comment by LLUMOAI** [comment]
Email [education@github.com](mailto:education@github.com) with your account details and the date your benefits were approved. Hope this helps :)

**Comment by asishkumar-gouda** [comment]
Hii @NagbhushanPai  you try contacting below handles .

To contact GitHub about your benefits, use the following methods:

1. **GitHub Support Portal**: Submit a request [here](https://support.github.com/request).
2. **Email**: Contact support at support@github.com.
3. **Community Forum**: Ask for help on the [GitHub Community Forum](https://github.community/).
4. **Twitter**: Reach out to [@GitHubHelp](https://twitter.com/githubhelp).
5. **GitHub Education (if applicable)**: Use the [Education Contact Form](https://support.github.com/contact/education).

Also,  Email [education@github.com](mailto:education@github.com)

**Comment by davevad93** [comment]
Hi @NagbhushanPai ,

If it says `"Your benefits will be available soon"` it means that you still have to wait for the benefits to become available, the status of the application will change to:

`Congrats! Your request for @NagbhushanPai was approved and your benefits are now available`

Don't worry, it may take a few days or in some cases a little longer, but it usually only takes a few days.
